### PR TITLE
Show all authors in the news card

### PR DESCRIPTION
**Pull request by SaeedNoshadi89**
Related to #409

### CODE REVIEW

File: core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt [hunk 1]
```diff
@@ -156,12 +156,17 @@ fun NewsResourceAuthors(
     authors: List<Author>
 ) {
     if (authors.isNotEmpty()) {
-        // Only display first author for now
-        val author = authors[0]
+        // display all authors
+        val authorName = StringBuilder()
+        authors.forEachIndexed { index, author ->
```
Comment (mmoczkowski) [inline core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt:161]: Thank you for your contribution Saeed. Could you replace this logic with `joinToString` method? I believe it's more suitable here.

@dturner Are we fine with displaying more authors?

**Review by mmoczkowski — CHANGES_REQUESTED** [review]
Please use `joinToString` method to concatenate authors

File: core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt (hunk 1 above)
Comment (SaeedNoshadi89) [reply core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt:161]: Of course

Comment (SaeedNoshadi89) [reply core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt]: Done

**Comment by SaeedNoshadi89** [comment]
> Please use `joinToString` method to concatenate authors

Done

**Review by mmoczkowski — APPROVED** [review]
Wonderful, thank you again!

**Pull request merged**